turbo.json

```diff
@@ -277,6 +277,7 @@
     "ZOHOCRM_CLIENT_ID",
     "ZOHOCRM_CLIENT_SECRET",
     "ZOOM_CLIENT_ID",
-    "ZOOM_CLIENT_SECRET"
+    "ZOOM_CLIENT_SECRET",
+    "INTERCOM_SECRET"
```

Comment:
Can we sort this alphabetically?

Reply:
ah shit, didnt see this. maybe in a follow up PR?

Reply:
Nah, it's ok